### PR TITLE
Add-Pagination

### LINKED ISSUE #30: Add Pagination

### PR DESCRIPTION
**Pull request by diyorzz**

### CODE REVIEW

File: StaySphere.Api/StaySphere.Domain/Pagination/PaginationExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            int pageNumber) where T : EntityBase
+        {
+            var count = source.Count();
+            var items = source.Skip((pageNumber - 1) * pageSize)
+                .Take(pageSize)
+                .ToList();
+
+            return new PaginatedList<T>(items, count, pageNumber, pageSize);
+        }
+    }
```
Comment (SharifovDeveloper) [inline StaySphere.Api/StaySphere.Domain/Pagination/PaginationExtensions.cs]: Asinxron ishlidigan metodini ham qo'shib qo'ying.

**Review by Nabiddinov — DISMISSED** [review]
good

**Review by SharifovDeveloper — APPROVED** [review]
Looks good

File: StaySphere.Api/StaySphere.Domain/Pagination/PaginationExtensions.cs (hunk 1 above)
Comment (diyorzz) [reply StaySphere.Api/StaySphere.Domain/Pagination/PaginationExtensions.cs]: Qushilganku

**Review by Nabiddinov — APPROVED** [review]
good

**Comment by Nabiddinov** [comment]
good

**Comment by Nabiddinov** [comment]
good

**Pull request merged**